### ISSUE TITLE
[Segmented Control] Development

### ISSUE THREAD

**Issue by DanielleRameau** [sl-design-system/components#3504]
### Provide a general summary of the issue here

-

### 👤 Your name

DR

**Comment by a11ymiko** [comment]
**Accessibility requirements**
Segmented control should be unordered list of buttons. 

List should have possibility to add its accessible name by either `aria-label` or `aria-labelledby`. Accessible name should explain what can be controlled via segmented controls: e.g. ‘View’ or `Sort by`.

Each button should have `aria-current` attribute. Selected option should have `aria-current=’true’` and all the others should have `aria-current=’false’`.

User should place focus on first segmented control button with tab but to access other button left and right arrows should be used. So no matter how many buttons segmented control have it’ll always be only one tab stop.

Example:
```
<ul aria-label="View">
    <li>
           <button aria-current='true'> List </button>
    </li>
    <li>
           <button aria-current='false'> Grid </button>
    </li>
</ul>
```
